Platform/NXP/iMX6ULL_EVK_512MB/Library/iMX6BoardLib/iMX6BoardInit.c

```diff
@@ -265,40 +186,36 @@ VOID EnetInit ()
   ImxPadConfig(IMX_PAD_ENET1_TXD0      , IMX_PAD_ENET1_TXD0_CTL       );
   ImxPadConfig(IMX_PAD_ENET1_TXD1      , IMX_PAD_ENET1_TXD1_CTL       );
   ImxPadConfig(IMX_PAD_ENET1_TX_EN     , IMX_PAD_ENET1_TX_EN_CTL      );
-  ImxPadConfig(IMX_PAD_ENET1_REF_CLK   , IMX_PAD_ENET1_REF_CLK_CTL    );
   ImxPadConfig(IMX_PAD_ENET1_RX_ER     , IMX_PAD_ENET1_RX_ER_CTL      );
   ImxPadConfig(IMX_PAD_ENET1_RX_EN     , IMX_PAD_ENET1_RX_EN_CTL      );
   ImxPadConfig(IMX_PAD_ENET1_RXD0      , IMX_PAD_ENET1_RXD0_CTL       );
-  ImxPadConfig(IMX_PAD_ENET1_RXD1      , IMX_PAD_ENET1_RXD1_CTL       );
-  MmioWrite32 (IMX_IOMUXC_SNVS_BASE + 0x20, IMX_IOMUXC_ALT5);
-  // Enable ENET PLL, also required for PCIe
+  ImxPadConfig(IMX_PAD_ENET1_RXD1      , IMX_PAD_ENET1_RXD1_CTL       );
+
+  // Enable ENET PLL
```

Comment:
Are these ENET settings already configured in U-Boot? If so, it would be cleaner to remove this init code and rely on U-Boot's configuration.